pkg/controller/certificaterequests/acme/acme_test.go

```diff
@@ -111,6 +112,32 @@ func TestSign(t *testing.T) {
 		}),
 	)
 
+	// Build root RSA CA
+	rootPK, err := pki.GenerateRSAPrivateKey(2048)
+	if err != nil {
+		t.Fatal()
```

Comment:
suggestion(non-blocking): I suspect this case will never be hit, but still would be nice to add more detail on the error

```suggestion
		t.Fatalf("failed to generate private key for test: %v", err)
```